kore/test/Test/Kore/Builtin/Definition.hs

```diff
@@ -1747,6 +1751,46 @@ testModule =
             ]
         }
 
+testModuleWithTwoClaims :: ParsedModule
+testModuleWithTwoClaims =
+    Module
+        { moduleName = testModuleName
+        , moduleAttributes = Attributes []
+        , moduleSentences =
+            [ SentenceClaimSentence . SentenceClaim $
+                (SentenceAxiom
+                    { sentenceAxiomParameters = [SortVariable (testId "sv1")]
+                    , sentenceAxiomPattern =
+                        asParsedPattern
+                            $ PatternF.StringLiteralF
+                            $ Const (StringLiteral "a")
```

Comment:
You can also make a `TermLike` here with the convenient `mk*` constructors and call `Kore.Builtin.External.externalize`.